org.omg.kerml.xtext/src/org/omg/kerml/xtext/KerML.xtext

```diff
@@ -454,14 +423,14 @@ OwnedConjugation returns SysML::Conjugation :
 /* Feature Membership */
 
 FeatureMember returns SysML::FeatureMembership :
-	TypeMemberPrefix FeatureMemberFlags 
+	DefinitionMemberPrefix FeatureMemberFlags 
```

Comment:
The rename of TypeMemberPrefix to DefinitionMemberPrefix was done as sysml.xtext contained the exact same content with a different name and DefinitionMemberPrefix seemed the newer terminology, but the synchronization could happen the other way if necessary.

Reply:
"Type" is a KerML term. "Definition" is a SysML term. All Definitions are Types, but not all Types are Definitions. "TypeMemberPrefix" should be used in the KerML grammar, not "DefinitionMemberPrefix".